### PR TITLE
0.4.0 release

### PR DESCRIPTION
**Pull request by mapsam**
Includes changes from #109 

cc @CyanRook for review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,8 +38,11 @@ The two major use cases for this library are:
         Defaults to all geometry types.
     -   `options.dedup` **[String](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)** perform deduplication of features based on shared layers, geometry, IDs and matching
         properties. (optional, default `true`)
+    -   `options.basic-filters` **[Array](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Array)&lt;[String](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String), [Array](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Array)>?** and expression-like filter out Number or Boolean properties based on
+        the following condtions: `=, !=, <, <=, >, >=`. The first item must be the value "any" or "all" whether any or all filters
```
Comment (CyanRook) [inline README.md]: Slight phrasing change - 
an expression-like filter to include features with Numeric or Boolean properties that match the filters based on the following conditions:

Or something similar, I was having trouble parsing 'and expression-like filter out'

**Review by CyanRook — CHANGES_REQUESTED** [review]
Small phrasing clarification in the readme, but other than that all good.

File: README.md (hunk 1 above)
Comment (mapsam) [reply README.md]: Good catch. Used your phrasing 👍

**Review by CyanRook — APPROVED** [review]

**Review by springmeyer — APPROVED** [review]
Nice update!

**Pull request merged**